### DISCUSSION TITLE
update dcm4chee-2.17.2-mysql to dcm4chee-arc-5.26.1-mysql

### DISCUSSION THREAD

**Discussion by kollia** [dcm4che/dcm4chee-arc-light#3707] [General]
Dear dcm4chee development-team,
Dear Mr. Zeillinger!

A great project your dcm4chee dicom server!!!
@gunterze 
I working in the austrian univ. clinic of graz for prof. Sorantin, I do not reach you over email -> now i gave this medium an try.

Is it possible to upgrade the dcm4chee 2.17. to 5.26?
I searched inside the internet for versions between 2.18. and 5.1., but I cannot find any. 
when I update the mysql database with tables from dcm4che-2.18.
$> mysql -u *** -p pacsdb < dcm4chee-arc-5.26.1-mysql/sql/update-5.1-mysql.sql
I get the error:
ERROR 1054 (42S22) at line 2: Unknown column 'access_control_id' in 'field list'

maybe there are versions between 2.18. to 5.1.?
can anybody give me a hint?

**Comment by gunterze** [comment]
You may use a [RESTful Service](https://petstore.swagger.io/index.html?url=https://raw.githubusercontent.com/dcm4che/dcm4chee-arc-light/master/dcm4chee-arc-ui2/src/swagger/openapi.json#/IMPORT-RS/ImportInstances) of dcm4chee 5.x to  [Import Instances on Storage](https://github.com/dcm4che/dcm4chee-arc-light/wiki/Import-Instances-on-Storage) received by dcm4chee 2.x, creating corresponding DB records in the DB of dcm4chee 5.x.

Best greetings to Prof. Sorantin!

My e-mail: gunterze@protonmail.com

**Comment by kollia** [comment]
THX
first now I have to finish installation of 5.26.1
after that I import the old data, when I get problems I contact you

nice regards
Alexander Kolli
